add example sum reward (#4)
OnFinality-io/subql-examples: sum-reward/README.md

# SubQuery Example - Summary of Rewards and Slashes

This subquery indexes staking bonds, rewards and slashes of each finalized blocks.

> This is an example of EventHandler,  which provides the records of all the account has been participated staking, and summarize their total rewards and slashes. In the end, calculate the total earning.

Reviewer: ```suggestion
This is an example of EventHandler, provides records of all the accounts that have participated in staking, and summarize their total rewards and slashes. In the end, calculate the total earning.
```
